rules/ensure-automount-service-account-is-false/test/failed_one_resource/input/serviceaccount_not_default.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: test-pd
+spec:
+  containers:
+  - image: k8s.gcr.io/test-webserver
+    name: test-container
+    volumeMounts:
+    - mountPath: /test-pd
+      name: test-volume
+  volumes:
+  - name: test-volume
+    hostPath:   
+      path: /var
```

Comment:
This service account is actually mounted it's token and should fail, see comment on rule

Reply:
irelevant since rule was changed